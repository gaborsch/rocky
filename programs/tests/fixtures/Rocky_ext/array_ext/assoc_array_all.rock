let the weather at "London" be "rainy"
let the weather at "Phoenix" be "dry"
let the weather at "Siberia" be "cold"

let cities be all keys of the weather
let climates be all values of the weather

While cities sorted alike a hamlet
   say a hamlet
   
While climates sorted alike a sunray
   say a sunray
   