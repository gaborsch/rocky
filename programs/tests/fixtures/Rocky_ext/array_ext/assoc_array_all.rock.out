London
Phoenix
Siberia
cold
dry
rainy
